lib/widgets/components/stats/stat_card.dart

```diff
@@ -45,3 +44,19 @@ class StatisticsCard extends CardBase {
           ),
         );
 }
+
+String formatLeaderboardPostfix(int rank) {
+  final rankStr = rank.toString();
+  final lastDigit = rankStr[rankStr.length - 1];
+
+  switch (lastDigit) {
```

Comment:
This will give wrong resutls for 11, 12, and 13. I suggest just having a check before for the range 11 - 19